org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/documentLink/LSPPresentationReconciler.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Rogue Wave Software Inc. and others.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Michal Niewrzal‚ (Rogue Wave Software Inc.) - initial implementation
+ *  Angelo Zerr <angelo.zerr@gmail.com> - fix Bug 521020
+ *******************************************************************************/
+package org.eclipse.lsp4e.operations.documentLink;
+
+import org.eclipse.jface.text.IDocument;
+import org.eclipse.jface.text.ITextViewer;
+import org.eclipse.jface.text.reconciler.IReconciler;
+import org.eclipse.jface.text.reconciler.MonoReconciler;
+
+/**
+ * {@link IReconciler} implementation to Highlight Symbol (mark occurrences like).
+ */
+public class LSPPresentationReconciler extends MonoReconciler {
```

Comment:
Would you mind renaming to `LSPDocumentLinkPresentationReconciler` ?

Reply:
LSPDocumentLinkPresentationReconciler means that it will take care only about document link. My idea is was to manage sematntic higlighting to avoid creating a new Thread reconciler for that. What do you think about that?

Reply:
I would rather have different reconcilers for different operations.

Reply:
ok fixed